install/terraform/gcp/cluster_node_pools_gpu_l4.tf

```diff
@@ -15,7 +15,8 @@ locals {
 
 # The L4 GPU does not support node autoprovisioning so precreating 0 size nodepool
 resource "google_container_node_pool" "g2-standard-4" {
-  name = "g2-standard-4"
+  name  = "g2-standard-4"
+  count = var.attach_gpu_nodepools ? 1 : 0
```

Comment:
Just a consideration: L4s are the GPU flavor we use a ton currently so g2-standard-4s may be worth always deploying. I think deploying the smallest g2 and none of the others would still be a solid 10 minutes faster provisioning time while allowing things like our quickstart to work.

Reply:
Are you thinking we get rid of this flag and just always create a single L4 node pool?

Reply:
I should have fleshed out my thought a little better here:

I think we should **always** deploy one node pool of a `g2-standard-4` (e.g. don't provision behind a var + conditional). Each of the other `g2-standard-*` node pools should be controlled by a variable + conditional like you have here. Along with that, we should rename `attach_gpu_nodepools` to something like `create_all_l4_gpu_nodepools` and default it to `false`.

Reply:
lets leave it as a variable

Reply:
Created and issue to track: https://github.com/substratusai/substratus/issues/159